src/CommunityToolkit/Xamarin.CommunityToolkit/build/PreserveXamarinCommunityToolkit.fs

```diff
@@ -0,0 +1,6 @@
+// <auto-generated/>
+
+open Xamarin.CommunityToolkit.Helpers
+
+[<assembly: PreserveToolkit>]
+do ()
```

Comment:
I guess "do" can be omitted

Reply:
Can it? Compilation fails without it:

![image](https://user-images.githubusercontent.com/6609929/112731116-1bad0400-8f3e-11eb-828d-d6c42758cad6.png)

Reply:
Also SO answer does it with `do`:
https://stackoverflow.com/a/2269735/4312132

Reply:
Ohhh, my bad. I proposed an incorrect fs file. Let me update it